aspnetcore/fundamentals/host/hosted-services/samples/3.x/BackgroundTasksSample/Services/QueuedHostedService.cs

```diff
@@ -64,10 +59,10 @@ public override async Task StopAsync(CancellationToken stoppingToken)
         {
             _logger.LogInformation("Queued Hosted Service is stopping.");
 
-            _shutdown.Cancel();
-
             await Task.WhenAny(_backgroundTask, 
```

Comment:
#7640 removed Task.WhenAny because the code has been extracted into Backgroundservice. Why reintroduce it?

Reply:
Yes, and this clears up that Host shutdown error, too.